ivy/functional/backends/paddle/statistical.py

```diff
@@ -72,7 +72,7 @@ def std(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.std(x, axis=axis, unbiased=True, keepdim=keepdims)
```

Comment:
you need to implement the correction parameter, the unbiased parameter in paddle sets correction = 1, but it needs to be a float, check the torch backend implementation as it's the closest one to paddle